core/code/hooks.js

```diff
@@ -64,9 +64,11 @@
 window._hooks = {}
 window.VALID_HOOKS = []; // stub for compatibility
 
+var isRunning = 0;
```

Comment:
If we assume that `removeHook` calls are rare, we can skip this extra `isRunning` check.

This would save a little bit performance at the cost of a 'splice' in every `removeHook`

Reply:
> This would save a little bit performance

Extremely little bit.
I'd rather accept it with another reason: for the sake of simplicity.

But here we have pretty same code as Leaflet, so I prefer to keep it closer to 'upstream'.